src/SFA.DAS.EmployerFinance.AcceptanceTests/Features/LateAccounts/Late-accounts.feature

```diff
@@ -0,0 +1,185 @@
+﻿Feature: Late Accounts Feature
```

Comment:
I don't think this feature needs its own folder.

Reply:
done